recipes/tyssue/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "tyssue" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "3787cd9bb9292f9ecb8d5a91db8706bfe95845f011b37f2890ece425270aae8d" %}
+
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/DamCB/tyssue/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
+    - scipy
+  run:
+    - python
+    - numpy
+    - scipy
+    - matplotlib
+    - pandas
+    - pytables
+    - jupyter
+    - notebook
```

Comment:
vispy is listed a requirement in: 

https://github.com/DamCB/tyssue/blob/master/doc/requirements.txt